LDAP-primer.md

```diff
@@ -41,10 +41,11 @@ effectively. The principal difficulty with gaining that knowledge lies
 in learning a sizeable set of interrelated concepts using rather alien
 terminology, stemming from LDAP's roots in the X.500 standardization
 effort. The IETF/ISO standard wars of the mid- to late 1980s are
-now forgotten, but one can glimpse some of the reasons for internet's
+now forgotten, but one can glimpse some of the reasons for the IETF's
 decisive victory by perusing any available X.500 document and noting
 the impenetrable text combined with a jungle of references to other
-such documents.
+such documents. Though ironically we now have a jungle of LDAP standards
+all referencing each other!
 
```

Comment:
I'm leaving out "Though ironically..." since I believe that LDAP standards have a slightly different pathology which can't be pithily summarized.